server.js

```diff
@@ -38,6 +38,23 @@ const app = express();
 const port = process.env.QUIZZARD_PORT || 8000;
 
 /* Pre-compiled Pug views */
+const studentTable = pug.compileFile('views/account-table.pug');
```

Comment:
Why are these back?

Reply:
My bad I goofed lol. I've removed them and added Pug to the name of my consts